_lifeofinternatpuppet/post_6.md

```diff
@@ -11,22 +11,17 @@ tags:
 
 Hi everyone!
 
-This week I have nothing much to say!
-As always the week started off with a 1:1 session with Norman.
-I continued to work on my Learning VM [Quest Guide](https://github.com/puppetlabs/puppet-quest-guide) after completing the VIM tutorial last week.
+This week I have nothing much to say! As always the week started off with a 1:1 session with Norman. I continued to work on my Learning VM Quest Guide after completing the VIM tutorial last week.
 
 I also started to look into doing some recap sessions on the traning received so far. Some of the topics I was looking at were Litmus and Running Tests.
 
-I did another session on the tag and release process with [Daiana](https://github.com/daianamezdrea)
```

Comment:
Did you mean to update this file? It looks like doing so removes some links?

Reply:
Yes, this file needs to be reviewed,  I am not sure why the links are getting removed. I have replied to your email. Please check it out for more clarity and information. I will wait for your reply

Reply:
https://github.com/puppetlabs/iac/pull/132/files#diff-61d588f92c0d7229e9e26c0c58c01ca9bd2b84511c272c4f0476cb4b5397c141 

This is the file (link given above) which needs to be updated, go to files changed and then jump to this file. The file is called post_7_and_post_8.md. Refer to Triage channel on slack and the life of intern channel. Details can be found there.

Reply:
Please ignore the post_6.md file.

Reply:
We can't ignore this file, these changes _will_ be made if we merge this PR as-is.